helix-term/src/ui/editor.rs

```diff
@@ -327,8 +328,30 @@ impl EditorView {
         let info: Style = theme.get("info");
         let hint: Style = theme.get("hint");
 
+        let line_added: Style = theme.get("diff.added");
+        let line_removed_above: Style = theme.get("diff.removed.above");
+        let line_removed_below: Style = theme.get("diff.removed.below");
+        let line_modified: Style = theme.get("diff.modified");
+
         for (i, line) in (view.first_line..last_line).enumerate() {
             use helix_core::diagnostic::Severity;
+            if let Some(line_changes) = doc.vcs_line_changes() {
```

Comment:
To address #405, maybe here we could abstract it to simply be `doc.line_changes()` . It doesn't need to be done in this PR however.

Reply:
Good point, though I agree that it is probably outside of the scope of this PR.